Update change-the-default-language-for-greetings-and-emails.md (#4675)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/change-the-default-language-for-greetings-and-emails.md

  
- **The languages that are available to you are determined by the location of your organization**. For example, if your organization is located in the United States, you can set the default language to English or Spanish. If your organization is located in Canada, you can choose between English and French. For a list of supported languages, see [Languages for voicemail greetings and messages from Skype for Business](languages-for-voicemail-greetings-and-messages.md).
    

> - **Changing languages for individual user's voicemail greeting and voicemail messages.** You can change the preferred lanaguage for users, which will change the language of the their voicemail greeting and voicemail messages sent to their Outlook mailbox. For instruction please see [How to set language and region settings for Office 365] (https://docs.microsoft.com/en-us/office365/troubleshoot/access-management/set-language-and-region). 

Reviewer: ```suggestion
- **Changing languages for individual user's voicemail greeting and voicemail messages.** You can change the preferred language for users, which will change the language of their voicemail greeting and voicemail messages sent to their Outlook mailbox. For instructions, please see [How to set language and region settings for Office 365] (https://docs.microsoft.com/office365/troubleshoot/access-management/set-language-and-region). 
```

    
    > [!NOTE]

>     > Users can change their own greeting language through their settings after they sign in by following instructions found in [Change your display language and time zone in Microsoft 365 for Business] (https://support.office.com/en-us/article/change-your-display-language-and-time-zone-in-microsoft-365-for-business-6f238bff-5252-441e-b32b-655d5d85d15b?ui=en-US&rs=en-US&ad=US)

Reviewer: ```suggestion
    > Users can change their own greeting language through their settings after they sign in by following instructions found in [Change your display language and time zone in Microsoft 365 for Business] (https://support.office.com/article/change-your-display-language-and-time-zone-in-microsoft-365-for-business-6f238bff-5252-441e-b32b-655d5d85d15b?ui=en-US&rs=en-US&ad=US).
```

> - **Do you want to record your outgoing voicemail message?** See [Check Skype for Business voicemail and options](https://support.office.com/article/2deea7f8-831f-4e85-a0d4-b34da55945a8). For Microsoft Teams - Users can change their voicemail settings from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f)

Reviewer: ```suggestion
- **Do you want to record your outgoing voicemail message?** See [Check Skype for Business voicemail and options](https://support.office.com/article/2deea7f8-831f-4e85-a0d4-b34da55945a8). For Microsoft Teams - Users can change their voicemail settings from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f).
```

> - **Do you want to change the voicemail prompt language?** For Skype for Business -  [https://mysettings.lync.com/voicemail](https://mysettings.lync.com/voicemail) and choose a new language under **Prompt Language**. For Microsoft Teams - Users can change their voicemail greeting from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f)

Reviewer: ```suggestion
- **Do you want to change the voicemail prompt language?** For Skype for Business - go to the [User Settings Portal](https://mysettings.lync.com/voicemail) and choose a new language under **Prompt Language**. For Microsoft Teams - Users can change their voicemail greeting from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f).
```
